src/styles/index.scss

```diff
@@ -1,3 +1 @@
-body {
-  margin: 0;
-}
+@import './main';
```

Comment:
Ensure that the `main.scss` file exists in the correct directory. Also, remember to compile your SCSS files into a CSS file for use in your HTML.